src/utils/formatDate.js

```diff
@@ -0,0 +1,9 @@
+export function formatDate(date) {
+  const day = String(date.getDate()).padStart(2, "0");
+  const month = String(date.getMonth() + 1).padStart(2, "0");
+  const year = String(date.getFullYear());
+
+  const formattedDate = `${day}.${month}.${year}`;
+
+  return formattedDate;
```

Comment:
```suggestion
  return date.toLocaleDateString('en-GB').replace(/\//g,'.')
```

Can you try this?

Reply:
@fza-wtag vai, using `date.toLocaleDateString('en-GB').replace(/\//g,'.')` actually shows the year of the date in 4 digits (According to design it should be 2 digits).

However, I have changed my previous code. Now it uses builtin Intl object for formatting the dates.